### PR TITLE
bootstrap table classes added to all tables

### PR DESCRIPTION
**Pull request by murex971**
Signed-off-by: Nupur Agrawal <nupur202000@gmail.com>

### Description

Applies bootsrap table classes to all the tables and provide an overall consistency across tables.

Before submitting pull request, please review the following checklist:

- [ ] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [ ] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [ ] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [ ] Every commit has a descriptive commit message.
- [ ] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [ ] Any new functionality is covered by tests.

### CODE REVIEW

**Comment by mauriciofauth** [comment]
@sudo-bot Deploy PR

**Comment by mauriciofauth** [comment]
> Applies bootsrap table classes to all the tables and provide an overall consistency across tables.

You commented that you added the Bootstrap classes in **all** the tables, but only the tables of the server pages received the classes, the database pages and the table pages are still missing. Is this pull request a work in progress?

**Comment by murex971** [comment]
> > Applies bootsrap table classes to all the tables and provide an overall consistency across tables.
> 
> @murex971 You commented that you added the Bootstrap classes in **all** the tables, but only the tables of the server pages received the classes, the database pages and the table pages are still missing. Is this pull request a work in progress?

yeah, it is still in progress.

File: templates/server/plugins/index.twig [hunk 1]
```diff
@@ -7,52 +7,45 @@
 <div id="plugins_plugins">
   <div id="sectionlinks" class="row no-gutters">
     <div class="col-12">
-      {% for type in plugins|keys %}
-        <a class="btn btn-primary" href="#plugins-{{ clean_types[type] }}">
-          {{ type }}
+      {% for plugin_type in plugins|keys %}
```
Comment (mauriciofauth) [inline templates/server/plugins/index.twig]:
```suggestion
      {% for type in plugins|keys %}
```

File: templates/server/plugins/index.twig [hunk 2]
```diff
@@ -7,52 +7,45 @@
...
+        <a class="btn btn-primary" href="#plugins-{{- plugins_type_clean[plugin_type] }}">
```
Comment (mauriciofauth) [inline templates/server/plugins/index.twig]:
```suggestion
        <a class="btn btn-primary" href="#plugins-{{ clean_types[type] }}">
```

File: templates/server/plugins/index.twig [hunk 3]
```diff
@@ -7,52 +7,45 @@
...
+            {{ plugin_type }}
```
Comment (mauriciofauth) [inline templates/server/plugins/index.twig]:
```suggestion
            {{ type }}
```

File: templates/server/plugins/index.twig [hunk 4]
```diff
@@ -7,52 +7,45 @@
...
   </div>
+
   {% for type, list in plugins %}
     <div class="responsivetable row">
       <div class="col-12">
-        <table class="data_full_width" id="plugins-{{ clean_types[type] }}">
+        <table class="data_full_width table table-responsive table-hover table-striped" id="plugins-{{ clean_types[type] }}">
           <caption class="tblHeaders">
-            {{ type }}
+            {{ plugin_type }}
```
Comment (mauriciofauth) [inline templates/server/plugins/index.twig]:
```suggestion
            {{ type }}
```

File: templates/server/plugins/index.twig [hunk 5]
```diff
@@ -7,52 +7,45 @@
...
+              <th scope="col">{% trans 'Plugin' %}</th>
+              <th scope="col">{% trans 'Description' %}</th>
+              <th scope="col">{% trans 'Version' %}</th>
+              <th scope="col">{% trans 'Author' %}</th>
+              <th scope="col">{% trans 'License' %}</th>
             </tr>
           </thead>
           <tbody>
-            {% for plugin in list %}
+            {% for plugin in plugin_list %}
```
Comment (mauriciofauth) [inline templates/server/plugins/index.twig]:
```suggestion
            {% for plugin in list %}
```

File: templates/server/plugins/index.twig [hunk 6]
```diff
@@ -7,52 +7,45 @@
...
-                      {% if plugin.status == 'INACTIVE' %}
-                        {% trans 'inactive' %}
-                      {% elseif plugin.status == 'DISABLED' %}
-                        {% trans 'disabled' %}
-                      {% elseif plugin.status == 'DELETING' %}
-                        {% trans 'deleting' %}
-                      {% elseif plugin.status == 'DELETED' %}
-                        {% trans 'deleted' %}
-                      {% endif %}
+                      {% trans 'disabled' %}
```
Comment (mauriciofauth) [inline templates/server/plugins/index.twig]:
```suggestion
                      {% if plugin.status == 'INACTIVE' %}
                        {% trans 'inactive' %}
                      {% elseif plugin.status == 'DISABLED' %}
                        {% trans 'disabled' %}
                      {% elseif plugin.status == 'DELETING' %}
                        {% trans 'deleting' %}
                      {% elseif plugin.status == 'DELETED' %}
                        {% trans 'deleted' %}
                      {% endif %}
```

File: templates/server/plugins/index.twig [hunk 7]
```diff
@@ -7,52 +7,45 @@
...
                   {% endif %}
                 </th>
-                <td>{{ plugin.description }}</td>
-                <td>{{ plugin.version }}</td>
-                <td>{{ plugin.author }}</td>
-                <td>{{ plugin.license }}</td>
+                <td>{{ plugin.plugin_description }}</td>
+                <td>{{ plugin.plugin_type_version }}</td>
+                <td>{{ plugin.plugin_author }}</td>
+                <td>{{ plugin.plugin_license }}</td>
```
Comment (mauriciofauth) [inline templates/server/plugins/index.twig]:
```suggestion
                <td>{{ plugin.license }}</td>
```

File: templates/server/plugins/index.twig [hunk 8]
```diff
@@ -7,52 +7,45 @@
...
                     </small>
                   {% endif %}
                 </th>
-                <td>{{ plugin.description }}</td>
-                <td>{{ plugin.version }}</td>
-                <td>{{ plugin.author }}</td>
-                <td>{{ plugin.license }}</td>
+                <td>{{ plugin.plugin_description }}</td>
+                <td>{{ plugin.plugin_type_version }}</td>
+                <td>{{ plugin.plugin_author }}</td>
```
Comment (mauriciofauth) [inline templates/server/plugins/index.twig]:
```suggestion
                <td>{{ plugin.author }}</td>
```

File: templates/server/plugins/index.twig [hunk 9]
```diff
@@ -7,52 +7,45 @@
...
+                      {% trans 'disabled' %}
                     </small>
                   {% endif %}
                 </th>
-                <td>{{ plugin.description }}</td>
-                <td>{{ plugin.version }}</td>
-                <td>{{ plugin.author }}</td>
-                <td>{{ plugin.license }}</td>
+                <td>{{ plugin.plugin_description }}</td>
+                <td>{{ plugin.plugin_type_version }}</td>
```
Comment (mauriciofauth) [inline templates/server/plugins/index.twig]:
```suggestion
                <td>{{ plugin.version }}</td>
```

File: templates/server/plugins/index.twig [hunk 10]
```diff
@@ -7,52 +7,45 @@
...
-                      {% endif %}
+                      {% trans 'disabled' %}
                     </small>
                   {% endif %}
                 </th>
-                <td>{{ plugin.description }}</td>
-                <td>{{ plugin.version }}</td>
-                <td>{{ plugin.author }}</td>
-                <td>{{ plugin.license }}</td>
+                <td>{{ plugin.plugin_description }}</td>
```
Comment (mauriciofauth) [inline templates/server/plugins/index.twig]:
```suggestion
                <td>{{ plugin.description }}</td>
```

**Review by MauricioFauth — CHANGES_REQUESTED** [review]

**Pull request closed without merging**